### PR TITLE
13 data points in 3d space

### PR DESCRIPTION
**Pull request by Jesster2829**
# What was the issue
Creating a way to map out data points relative to the axis, taking into account scale and length. Also 
removing the old prototype from the system was a priority in this ticket

# What was done and why
First I created a function that calculates the position a data point will take, allowing for easy calls as long as you have access to the axis's props. In the next sprint I want to get a quick issue ticket for converting the props into a type, that way we only need to pass in a single object instead of 4-5 parameters. 

I also had to fix the positioning of the Axis since they were not in the correct position due to the <group> they were inside of having been assigned a position as well. 

# How it was tested
First I had created a couple Unit tests to start the test driven development and to give myself an idea of where to start. After I had the function finished and the data points also properly being displayed in what seemed like the right coordinates, I ran some calculations by had to figure out the position myself on paper and the math checked out. Further, I changed the scaling and the length of the Axis to see how the points would adjust and it still works. It still cannot handle rotation since that isnt taken account into the formula so that will have to be a future ticket.

### CODE REVIEW

File: my-webxr-app/src/App.tsx
Comment (qpb948) [inline my-webxr-app/src/App.tsx:121]: These are the data points from before Aesha’s merge right? Did you forget to merge your branch with ID3 branch before merging to ID3?

**Review by qpb948 — COMMENTED** [review]
Looks good.
Just the one comment on the data points. Whether you merged release/ID3 into yours after #26 was merged in.

File: my-webxr-app/src/App.tsx
Comment (matthew-buglass) [inline my-webxr-app/src/App.tsx:29]:
```suggestion
```

File: my-webxr-app/src/App.tsx
Comment (Jesster2829) [reply my-webxr-app/src/App.tsx:121]: I merged it with ID3 a couple hours ago, but did not do it just before the merge. But yes this was them before the datapoint changes that aesha made. I believe that her changes would not affect mine, since meshProps{{position:}} is still a thing and thats all my change really touches

**Comment by matthew-buglass** [comment]
I'm going to hold off my review until the merge conflicts are resolved.

**Review by qpb948 — APPROVED** [review]
Good unit tests and implementation of createPosition function.
Will need to plan for rotation of axes.

**Pull request merged**
